compiler/AST/checkAST.cpp

```diff
@@ -247,6 +247,7 @@ void checkPrimitives()
      case PRIM_CAPTURE_FN:
      case PRIM_CREATE_FN_TYPE:
      case PRIM_STRING_COPY:
+     case PRIM_MEMCPY:
```

Comment:
I might just not be seeing another way to do it, but I need to add calls to memcpy after function resolution runs - but in a pass before code generation. I don't know how to do that without making a primitive. Do you?